rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/action/TbCreateAlarmNode.java

```diff
@@ -209,4 +209,8 @@ private AlarmSeverity processAlarmSeverity(TbMsg msg) {
         return severity;
     }
 
+    long currentTimeMillis() {
+        return System.currentTimeMillis();
+    }
```

Comment:
: )

Reply:
It is what it is :)
I am not a fan of this, but I know no other way to test timestamp like in this case
If you know one, please let me know!